frontend/public/components/events.jsx

```diff
@@ -251,6 +250,22 @@ class EventStream extends SafetyFirst {
   }
 
   static filterMessages (messages, {kind, category, filter, textFilter}) {
```

Comment:
I know that this is not new but shouldn't this be `filterEvents` ? since we are now filtering based on name or message ?
Also the `messages` arg are in fact `events.
Don't want to bikeshed, but was a bit confused how the filter is getting the messages